src/runtime/audience-action-local-flow.ts

```diff
@@ -44,6 +47,8 @@ export interface AudienceActionLocalParams {
   monetizationFunction?: () => void;
 }
 
+const TYPE_REWARDED_AD = 'TYPE_REWARDED_AD';
```

Comment:
nit: Can we define it in a centralized place? 
It would be awesome if you could update other parts, e.g., https://github.com/subscriptions-project/swg-js/blob/09df5f3068bcbff8d010eccfe7e679035fd09844/src/runtime/audience-action-flow.ts#L67. Thanks!

Reply:
Done.

Reply:
Could use more followup cleanup, but will keep that for another cl.